docs/resources/Tutorials/how-to-annotate-a-genome.md

```diff
@@ -0,0 +1,136 @@
+
+1. Deciding which tools are best suited for generating a high-quality annotation, and 
+2. Actually running those tools on an HPC cluster.
+
+Here, we provide a brief overview of the three dominant flavors of genome annotation represented by current state-of-the-art tools, provide guidance for picking the right tools, and briefly describe and point to [Snakemake](https://snakemake.readthedocs.io/en/stable/) worfklows that can, with a modest amount of parameter specification, automate the generation of genome annotations in gff/gtf format. If you are unfamiliar with this file format, take a look at [this file format definition](http://useast.ensembl.org/info/website/upload/gff.html) found at *Ensembl*.
+
+## What do we mean by genome annotation?
+
+The term _**"genome annotation"**_ is typically used to refer to two discrete bioinformatics tasks. The first of these is structural annotation, namely the prediction of the genomic intervals comprised of functional genome features: genes, transcripts associated with those genes, the exons comprising those transcripts, and for protein-coding transcripts their coding sequences (CDS) and untranslated regions (UTRs). The second meaning of genome annotation is functional annotation, namely assigning gene symbols, Gene Ontology terms, and putative functions with respect to molecular phenotype. This tutorial is focused on the first of these meanings, namely choosing and running tools to predict functional features on a genome assembly.
+
+## Three major approaches to genome annotation
+
+### 1. Model-based approaches
+
+The current state-of-the-art tools for genome annotation can be divided into three categories. The first of these is the model-based approach that uses a Hidden Markov Model (HMM) to scan a genome sequence, classify particular segments as protein-coding, group these coding sequence (*i.e.* CDS) intervals into transcripts, and to aggregate those transcripts into genes. Among the several tools that use this approach, the most well-known and widely used is [AUGUSTUS](https://github.com/Gaius-Augustus/Augustus), the first version of which was published in 2003. Widely used optimized versions of AUGUSTUS are implemented in [BRAKER](https://github.com/Gaius-Augustus/BRAKER), which can take extrinsic evidence in the form of protein sequences or RNA-seq data which are aligned to the genome to provide hints as to where splicing takes place, and to parameterize the underlying HMMs; BRAKER also generates annotations with [Genemark](https://github.com/gatech-genemark). Model-based approaches typically only predict protein-coding features, such that non-coding RNAs and UTRs are not included in their output. 
```

Comment:
does 'extrinsic' here imply this is data from another species? or data from the same species but not in the assembly?